src/chrome/content/rules/Technical_University_Munich.xml

```diff
@@ -45,37 +39,68 @@
 
 	* Secured by us
 
+	Expired certificates:
+		- ias.in
+		- ias.cs.tum.edu
+		- ias.informatik.tu-muenchen.de
 -->
 <ruleset name="Technical University Munich (partial)">
 
-	<target host="*.tum.de" />
-	<target host="ias.cs.tum.edu" />
-	<target host="*.tu-muenchen.de" />
-
+	<target host="www.tu-muenchen.de" />
+
+	<target host="www.tum.de" />
+	<target host="www.lehre.tum.de" />
+	<target host="www.lehren.tum.de" />
+	<target host="www.piwik.tum.de" />
+	<target host="mediatum.ub.tum.de" />
+	<target host="mailmanbroy.informatik.tu-muenchen.de" />
+	<target host="adm9.in.tum.de" />
+	<target host="www.in.tum.de" />
+	<target host="www21.in.tum.de" />
+	<target host="www.lrr.in.tum.de" />
+	<target host="www.sec.in.tum.de" />
+	<target host="isabelle.in.tum.de" />
 
 	<!--	Not secured by server:
 					-->
 	<!--securecookie host="^www\.sec\.in\.tum\.de$" name="^PHPSESSID$" /-->
 	<!--securecookie host="^mediatum\.ub\.tum\.de$" name="^PSESSION$" /-->
 
 	<securecookie host="^(?:in|(?:adm9|ias|sec|www)\.in|mediatum\.ub|www(?:\.lehren?)?)?\.tum\.de$" name=".+" />
-	<securecookie host="^ias\.cs\.tum\.edu$" name=".+" />
-	<securecookie host="^(?:www|ias\.informatik)\.tu-muenchen\.de$" name=".+" />
+	<securecookie host="^(?:www\.informatik)\.tu-muenchen\.de$" name=".+" />
```

Comment:
I added the target host, but I don't know how to remove the non-capturing groups. Sorry.